packages/runtime/container-runtime/src/test/runningSummarizer.spec.ts

```diff
@@ -117,7 +117,7 @@ describe("Runtime", () => {
                     summaryProposal,
                 };
                 mockDeltaManager.emit("op", {
-                    contents,
+                    data: JSON.stringify(contents),
```

Comment:
should we keep a test that verifies the contents property version keeps working until we fully remove it?

Reply:
I'd break it into these buckets:
1. Current service: This is not required, as (latest version of the) client handles (and validates) data property
2. UT: We need content to work simply because LTS branch testing (0.45) brings memorylicious driver, and with that - memorelicious service from 0.45 and that uses contets only, not data. So I had to make it work in this PR. So it's being tested (even though we do not care about such old version of service).
3. snapshot tests: they contain files in old formats. Snapshot tests do not care about summaries themselves, so even ignoring summary ack would work. 
4. Old files. Same as above, though we need to make sure clients works. I'd prefer for client not to see these ops at all, as client should recover (and we are testing that) when summary ack does not make it into a file. But due to # 2, we are properly processing them (and that's covered in that UT).

I feel fine not adding more tests, given that's very corner case and it's being tested (in terms of files working) through snapshots tests - as long as they keep working, the rest is not interesting.

Let me know if you disagree.